spark/src/main/java/org/apache/iceberg/spark/SparkTableUtil.java

```diff
@@ -305,6 +306,9 @@ private SparkTableUtil() {
    * For Parquet and ORC partitions, this will read metrics from the file footer. For Avro partitions,
    * metrics are set to null.
    *
+   * Note: certain metrics, like NaN counts, that are only supported by iceberg file writers but not file footers, will
```

Comment:
Javadoc needs a `<p>` on the empty line above if you want to start a new paragraph.